testsuite/pytests/test_status.py

```diff
@@ -76,7 +76,9 @@ def test_GetDefaults(self):
 
         nest.ResetKernel()
 
-        for model in nest.Models():
+        node_models = nest.GetKernelStatus("node_models")
+        synapse_models = nest.GetKernelStatus("synapse_models")
+        for model in node_models + synapse_models:
```

Comment:
```suggestion
        for model in nest.node_models + nest.synapse_models:
```
Or did I misunderstand something about this test?